docs/manual/manual_throttle.rst

```diff
@@ -0,0 +1,346 @@
+the Tuning tab in IQ Control Center, as shown below.
+
+.. figure:: ../_static/manual_images/throttle/throttle_max_volts.png
+    :align: center
+    :width: 60%
+    :alt: Max Volts Parameter
+
+    Max Volts Parameter in IQ Control Center
+
+Maximum Velocity
+#################
+The maximum velocity is used when the module is configured to use :ref:`throttle_velocity_mode` mode. The maximum velocity can be configured using the *Max Velocity* parameter under the Tuning 
+tab in IQ Control Center, as shown below.
+
+.. figure:: ../_static/manual_images/throttle/throttle_max_velocity.png
+    :align: center
+    :width: 60%
+    :alt: Max Velocity Parameter
+
+    Max Velocity Parameter in IQ Control Center
+
+.. _throttle_direction:
+
+Direction
+**********
+Another important configuration to consider before controlling a Vertiq module is the module’s spin direction. Throttle commands can be from -100% to 100%, 
+and the sign indicates the intended spin direction. Users can configure what direction a positive throttle command should be interpreted as by configuring 
+the direction of the module.
+
+The direction can be clockwise or counterclockwise. When the direction is clockwise, the module will spin clockwise when it receives a positive throttle 
+command and counterclockwise when it receives a negative throttle command. When the direction is counterclockwise, the module will spin counterclockwise 
+when it receives a positive throttle command and clockwise when it receives a negative throttle command.
+
+For example, if the module’s direction is set to clockwise and a 50% throttle command is received, the module will spin at 50% of its maximum value in 
+the clockwise direction. If a -50% throttle command is received, the module will spin at 50% of its maximum value in the counterclockwise direction.
+
+The direction can be set using the *Motor Direction* parameter in the General tab of the IQ Control Center as shown below. Note that this parameter specifies 
+whether the module is 2D or 3D as well as if the direction is clockwise or counterclockwise. The 2D and 3D selection is not important for determining what 
+direction the motor will spin when receiving a positive throttle command, only the clockwise or counterclockwise selection matters for that. The importance 
+of the 2D/3D distinction will be covered in the :ref:`throttle_mapping` section.
+
+.. figure:: ../_static/manual_images/throttle/throttle_motor_direction.png
+    :align: center
+    :width: 60%
+    :alt: Motor Direction Parameter
+
+    Motor Direction Parameter in IQ Control Center
+
+.. _throttle_mapping:
+
+Mapping to Throttle
+==============================
+
+As mentioned in the :ref:`throttle_sources` section above some throttle sources, namely IQUART and hobby protocols, send their throttle commands as a number from 0.0 to 1.0, 

```

Comment:
As mentioned in the :ref:`throttle_sources` section above, IQUART and hobby protocols send ...